tests/model_serving/model_server/utils.py

```diff
@@ -151,11 +151,11 @@ def verify_inference_response(
                 response = res[inference.inference_response_key_name]
                 if isinstance(response, list):
                     response = response[0]
-
-                response_text = response[inference.inference_response_text_key_name]
-                assert response_text == expected_response_text, (
-                    f"Expected: {expected_response_text} does not mathc response: {response_text}"
-                )
+                    if isinstance(response, dict):
```

Comment:
if this is inside the if block above, then no validation is taking place for the case where response is not a list, is that what we want?

Reply:
yes; we get to the exception

Reply:
What if it is not a dict?

Reply:
you reach the `else` on line 160

Reply:
How can the line 160 be executed in this case? The `if isinstance(response, dict):` part is inside the `if isinstance(response, list):` block, no?

Reply:
need to replace my glasses; thanks for catching. fixing